### ISSUE TITLE
How to use table detection pre-trained weights to train a custom dataset

### ISSUE THREAD

**Issue by Aditi81** [microsoft/table-transformer#54]
I have created a custom table detection dataset that has different class labels. Instead of training a model from scratch, I want to use the pre-trained weights for the table detection model trained on PubTables-1M. Please help me to proceed with this task.

**Comment by bsmock** [comment]
This should be possible with probably just a few changes to the code depending on how closely what you're trying to do matches what we did. You said you have a custom dataset for table detection with different class labels. So you have class labels besides "table" and "table rotated"?

**Comment by Aditi81** [comment]
In addition, how should I use the pre-trained weights for the table detection model trained on PubTables-1M?

**Comment by Atul997** [comment]
You can register your train and validation  dataset  using 
`register_coco_instances('dataset_train', path to json, path to images)`
`register_coco_instances('dataset_train', path to json, path to images)`
  `md = MetadataCatalog.get("dataset_train")`
 ` md.set(thing_classes=["class1", "class2", "class3"])`

You can use pretrained weights by
`python train.py --path to config  --weights pubtables-1m weights `
